client/state/themes/reducer.js

```diff
@@ -223,11 +223,13 @@ export const themeRequestErrors = createReducer(
  * @return {Object}        Updated state
  */
 export function queryRequests( state = {}, action ) {
+	let serializedQuery;
```

Comment:
This is a linting fix for `no-case-declarations`:

https://eslint.org/docs/rules/no-case-declarations